### PR TITLE
kompose: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added `kompose` - a tool to help users who are familiar with docker-compose move to Kubernetes.

### CODE REVIEW

File: pages/common/kompose.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# kompose
+
+> Kompose is a tool to help users who are familiar with docker-compose move to Kubernetes.
```
[the remark this answers is not in the document]
Comment (Aracki) [reply pages/common/kompose.md]: Yes, I agree.

File: pages/common/kompose.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Deploy your Dockerized application to a container orchestrator:
+
+`kompose up -f {{docker-compose.yml}}`
+
+- Delete instantiated services/deployment from kubernetes:
+
+`kompose down -f {{docker-compose.yml}}`
+
+- Convert a Docker Compose file into Kubernetes resources file:
```
[the remark this answers is not in the document]
Comment (Aracki) [reply pages/common/kompose.md]: Fixed!

File: pages/common/kompose.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# kompose
+
+> Kompose is a tool to help users who are familiar with docker-compose move to Kubernetes.
+
+- Deploy your Dockerized application to a container orchestrator:
```
[the remark this answers is not in the document]
Comment (Aracki) [reply pages/common/kompose.md]: Sorry I don't get this one.

[the remark this answers is not in the document]
Comment (agnivade) [reply pages/common/kompose.md]: Sorry, that syntax is common in code reviews. It means "substitute `your` with `a`"

File: pages/common/kompose.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (agnivade) [reply pages/common/kompose.md]: We don't need to repeat "Kompose is" again. The title clearly mentions it.

[the remark this answers is not in the document]
Comment (Aracki) [reply pages/common/kompose.md]: Fixed!

File: pages/common/kompose.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Aracki) [reply pages/common/kompose.md]: It makes sense. Similar like Vim's `%s/your/a`. Thanks. 🙂

**Comment by agnivade** [comment]
Thank you for your patience and making separate thoughtful commits !

**Comment by agnivade** [comment]
Actually commit 1 and 2 should be squashed. And then 3 and 4 can be rebased. But someone has to do it manually.

If you want to do it, please feel free :), otherwise this can be squashed.

**Pull request merged**